pkg/metric/nfs_stat_collector.go

```diff
@@ -443,6 +443,19 @@ func getNfsCapacityStat(pvName string, info nfsInfo, p *nfsStatCollector) ([]str
 	return nil, errors.New("capacity metrics from storage-monitor missing or invalid")
 }
 
+func isNewInfoVersion(info *nfsCapacityInfo) bool {
+	return info.UsedSizeInBytes != 0
+}
+
+func getTotalAndUsedSize(info *nfsCapacityInfo, isNewVersion bool) (total int64, used int64) {
+	if isNewVersion {
+		total, used = info.TotalSize, info.UsedSizeInBytes
```

Comment:
```suggestion
	if info.UsedSizeInBytes != 0 {
		total, used = info.TotalSizeInBytes, info.UsedSizeInBytes
```
Maybe we don't need the isNewVersion parameter?
We may not change the meaning of existing field, or we will break users that does not upgrade CSI.